src/parser/Parser.Class.spec.ts

```diff
@@ -73,12 +73,12 @@ describe('parser class', () => {
     it('does not allow "throw" to be defined as a local var', () => {
         const parser = Parser.parse(`
             sub main()
-                'not allowed to define throw as local var
+                'not allowed to define "throw" as local var
                 throw = true
             end sub
         `);
 
-        expect(parser.diagnostics[0]?.message).to.eql(DiagnosticMessages.cannotUseReservedWordAsIdentifier('throw').message);
+        expect(parser.diagnostics[0]?.message).to.eql(DiagnosticMessages.foundUnexpectedToken('=').message);
```

Comment:
Checking for `throw` token earlier in the process causes a more generic error to be thrown. It's not really worth refactoring the parser just to get a slightly better error message, and we will definitely still get an error message so it's fine to just change this test.